waypoints/build.boot

```diff
@@ -19,13 +19,18 @@
 (deftask package []
   (comp
    (download :url (format "https://github.com/imakewebthings/waypoints/archive/%s.zip" +lib-version+)
-             :checksum "88c1d53f9c0ac0eacc8487223558b317"
              :unzip true)
    (sift :move {#"^waypoints-\d+\.\d+\.\d+/lib/noframework\.waypoints\.js$"
                 "cljsjs/waypoints/development/waypoints.inc.js"
                 #"^waypoints-\d+\.\d+\.\d+/lib/noframework\.waypoints\.min.js$"
                 "cljsjs/waypoints/production/waypoints.min.inc.js"})
-   (sift :include #{#"^cljsjs"})
-   (deps-cljs :name "cljsjs.waypoints")
+   (deps-cljs :foreign-libs [{:file #"waypoints\.inc\.js"
+                              :file-min #"waypoints\.min\.inc\.js"
+                              :provides ["waypoints.js" "cljsjs.waypoints"]
```

Comment:
Looks like this library uses npm name `waypoints`: https://www.npmjs.com/package/waypoints

Provides and global-exports should match that name.